mangaki/mangaki/admin.py

```diff
@@ -141,27 +143,85 @@ def make_sfw(self, request, queryset):
 
     def merge(self, request, queryset):
```

Comment:
Please wrap this in a `transaction.atomic`. I'm not 100% confident this code handles correctly all the possible cases today, and I'm 100% sure it one day won't be updated for one modification or another. Having this in `atomic` allows us to prevent any modification from being done if the merge fails, instead of leaving the database in an intermediate state where a merge was only partially performed.